docs/DeveloperGuide.md

```diff
@@ -208,6 +208,37 @@ The following activity diagram summarizes what happens when a user executes a ne
 
 ![EditApplicantActivityDiagram](images/EditApplicantActivityDiagram.png)
 
+### MarkApplicant feature
+
+The `markapplicant` mechanism is facilitated by `AddressBook`. `MarkApplicant` extends `Command` class.
+`MarkApplicant` updates the status via creating a new applicant object with  the new status value
+and same original Applicant details.  This new Applicant object will replace the current Applicant object that is in the `AddressBook`.
+
+When the user wants to mark an applicant, the user will input `markapplicant` along with the index number of Applicant,
+and the new status value, which includes `pending`, `rejected`, and `accepted`
+
+For example, `markapplicant 1 s/rejected` will change the status of applicant in index 1 to rejected.
```

Comment:
Maybe you could also add that when a Applicant object is created, the initial value of the Applicant Status is set to "pending".

Reply:
I updated it!